api/s2n.h

```diff
@@ -15,12 +15,25 @@
 
 #pragma once
 
+#define S2N_GCC_VERSION (__GNUC__ * 10000 \
+                     + __GNUC_MINOR__ * 100 \
+                     + __GNUC_PATCHLEVEL__)
+
+#define S2N_GCC_VERSION_AT_LEAST(major, minor, patch_level) \
```

Comment:
This is pretty specific. Why not have a macro to convert to the single value and then callers can use their own operators.

Reply:
This was just moved from its original location in s2n_compiler.h